### ISSUE TITLE
Pseudo‐chilblain

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6778]
**Preferred term label:**
Pseudo‐chilblain

**Synonyms**
COVID toe (lay person)

**Definition (free text, please give PubMed ID)**
Acral areas of erythema with vesicles or pustules.
PMID:32348545 

Comment:
These lesions may resemble chilblains and have purpuric areas, affecting hands and feet.

**Parent term (use hpo.jax.org/app)**
Inflammatory abnormality of the skin HP:0011123

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Comment: This condition is termed pseudo-chilblain because of the similar appearance of these lesions to chilblains. Pseudochilblains may be observed in persons with COVID-19 without documented exposure to cold or other risk factors for chilblains. PMID:33479703

**Comment by pnrobinson** [comment]
Another synonym: chilblain-like lesion

**Comment by pnrobinson** [comment]
added
